project/Common.scala

```diff
@@ -18,7 +18,7 @@ object Common extends AutoPlugin {
 
   override lazy val projectSettings = reformatOnCompileSettings ++
     Dependencies.Common ++ Seq(
-    organization := "com.lightbend.akka",
+    organization := "com.lightbend.akka.management", // FIXME proposing this change, then we can have "com.lightbend.akka.management|discovery" etc; we'd do management.commercial for example later
```

Comment:
WDYT?
Though akka discovery maybe should be not under management?

Reply:
Isn't this going to be problematic with releasing, we'll need permissions for an extra group name etc.?
It's not like it will conflict with another akka-management or akka-discovery modules?

Reply:
We have permissions to com.lightbend.akka so we can release under the com.lightbend.akka.management as well